src/site/content/en/connect/index.11tydata.js

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
My hunch is we won't need to use pagination (plus it would make filtering hard). I think you can remove this file.